lib/kotlin_detekt/plugin.rb

```diff
@@ -180,7 +180,16 @@ def send_inline_comment(issues)
             added_lines = parse_added_line_numbers(git.diff[filename].patch)
             next unless added_lines.include? line
           end
-          send(level == "warning" ? "warn" : "fail", r.get("message"), file: filename, line: line)
+          reason = r.get("message")
+          rule = r.get("source")
```

Comment:
It would be nice if we can link to the rule but the xml doesn't have that information